elixir/apps/web/lib/web/live/sites/new_token.ex

```diff
@@ -273,9 +273,6 @@ defmodule Web.Sites.NewToken do
       "--sysctl net.ipv6.conf.all.forwarding=1",
       "--sysctl net.ipv6.conf.default.forwarding=1",
       "--device=\"/dev/net/tun:/dev/net/tun\"",
-      Enum.map(env ++ [{"FIREZONE_ENABLE_MASQUERADE", "1"}], fn {key, value} ->
```

Comment:
You removed all environment variables, instead just remove the `++ [{"FIREZONE_ENABLE_MASQUERADE", "1"}]`

Reply:
good catch!

Reply:
Ah, haha thanks! Looking at the code, I could have inferred that. I reads similar to Nix!